### ISSUE TITLE
Update documentation to indicate to update the templates upon spin upgrade

### ISSUE THREAD

**Issue by MChorfa** [fermyon/spin#1030]
Please see:
https://github.com/fermyon/spin/issues/615#issuecomment-1376073184
https://github.com/fermyon/spin/issues/615#issuecomment-1376125638

**Comment by vdice** [comment]
Thank you for the issue @MChorfa.  Want to make sure I'm understanding this correctly.

Were you thinking the docs addition here would be a note on the [Quickstart page](https://developer.fermyon.com/spin/quickstart#creating-a-new-spin-application-from-a-template) around the potential need to update local templates when the Spin CLI is upgraded (and perhaps showing an example command doing so)?

**Comment by MChorfa** [comment]
Yes, that is accurate. Either a note or an example command would work.
